jdaviz/components/viewer_data_select.vue

```diff
@@ -142,7 +142,7 @@ module.exports = {
           return false
         } else if (this.$props.viewer.reference === 'image-viewer' && item.type !== 'image') {
           return false
-        } else if (item.meta.mosviz_row !== undefined) {
+        } else if ((item.meta.mosviz_row !== undefined) || (item.meta.mosviz_row !== null)) {
```

Comment:
I fee like @kecnry should review this one since he did a lot of work with these menus.

Reply:
ah, reviewed before you changed this to have both and the or... shouldn't it be and logic though? 🤔

Reply:
Whoops, you're right. It should be `&&`

Reply:
Ok done in https://github.com/spacetelescope/jdaviz/pull/1798/commits/eaca4f6cc279682c5bb2e426e62c219e9e91ab95